app/views/shared/export/_plan.erb

```diff
@@ -40,7 +40,7 @@
             <% end %>
 
             <% answer = @plan.answer(question[:id], false) %>
-            <% blank = answer.present? ? answer.text.gsub(/<\/?p>/, '').gsub(/<br\s?\/?>/, '').chomp.blank? : true %>
+            <% blank = (answer.present? && !answer.text.blank?) ? answer.text.gsub(/<\/?p>/, '').gsub(/<br\s?\/?>/, '').chomp.blank? : true %>
```

Comment:
@briri, please consider is_valid? instead (e.g. https://github.com/DMPRoadmap/roadmap/blob/development/app/models/answer.rb#L75) since answers for an option_based question the text is purely a comment.